### PR TITLE
Visuals.j: Document more natives

### PR DESCRIPTION
**Pull request by Luashine**
I've lost track which ones :)

However here's the link for Display...Text...Player and how the positions change: [DisplayTextToPlayer-position](https://github.com/Luashine/wc3-test-maps/blob/master/DisplayTextToPlayer-position/DisplayTextToPlayer-position.md). I think I'll do more of these "out-of-tree", following the presentation vs semantics argument.

### CODE REVIEW

File: visuals.j [hunk 1]
```diff
@@ -251,27 +360,151 @@ native SetTextTagFadepoint          takes texttag t, real fadepoint returns noth
...
 
+/**
+Toggles minimap creep display.
+
+The creep camps are shown as green/orange/red circles by default and there's a button
+next to the minimap to toggle it while playing (hotkey: Alt+R).
+
+@note See: `GetCreepCampFilterState`, `SetAllyColorFilterState`
+
+@async
```
Comment (lep) [inline visuals.j:422]: Does it make sense to use `@async` here? As the native doesn't return a value.

Comment (Luashine) [reply visuals.j]: You are right, probably a copy-paste mistake

**Comment by lep** [comment]
Thank you again a ton. It's very nice to see that we get genuine new findings in here.
I took the liberty and squashed your commits a bit to remove the merge and fixup commits so I have to close this PR by hand.

**Comment by Luashine** [comment]
It's alright. I am experimenting a little more with git this time and didn't want to force-push with new history although its fine in other places.

**Pull request closed without merging**